src/js/src/components/notifications/NotificationItem.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="notification"
+       :class="notification.type">
+    <button class="notificationCloseBtn"
+            type="button"
+            @click="doDismiss(notification)">
+      X
+    </button>
+    <h2 v-if="notification.title">
+      {{ notification.title }}
+    </h2>
+    <p>
+      {{ notification.text }}
+    </p>
+  </div>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex'
+
+export default {
+  name: 'NotificationItem',
+ 
+  props: {
+    notification: {
+      type: Object,
+      required: true,
+      validator: function (obj) {
```

Comment:
Neat!

Reply:
But as we just discussed - the different props should perhaps be optional, and make room for a "custom" type error message.